Classes/person.rb

```diff
@@ -1,3 +1,4 @@
+require './Classes/corrector'
```

Comment:
- [ ] `corrector.rb`  is on the same folder as `person.rb`, so the right path to it is:
```ruby
require './corrector'
```
Kindly change it 😄

Reply:
Just Corrected that, LOL